.changeset/slimy-numbers-grab.md

```diff
@@ -0,0 +1,5 @@
+---
+'backend': major
```

Comment:
not sure about `major` here. It is just a bug fix that is not breaking any compatibility, should it be just a `patch`?

Reply:
Ok, done.